### PR TITLE
fixup drift from 50 release

### PR DESCRIPTION
**Pull request by coury-clark**
Fix drift in release branch by running go generate

## Test plan

N/A it's just go generate

### CODE REVIEW

File: internal/database/migration/shared/data/stitched-migration-graph.json [hunk 1]
```diff
@@ -9237,6 +9237,21 @@
 				],
 				"IsCreateIndexConcurrently": false,
 				"IndexMetadata": null
+			},
+			{
+				"ID": 1678832491,
+				"Name": "remove sg_jsonb_concat_agg",
+				"UpQuery": "-- Perform migration here.\n--\n-- See /migrations/README.md. Highlights:\n--  * Make migrations idempotent (use IF EXISTS)\n--  * Make migrations backwards-compatible (old readers/writers must continue to work)\n--  * If you are using CREATE INDEX CONCURRENTLY, then make sure that only one statement\n--    is defined per file, and that each such statement is NOT wrapped in a transaction.\n--    Each such migration must also declare \"createIndexConcurrently: true\" in their\n--    associated metadata.yaml file.\n--  * If you are modifying Postgres extensions, you must also declare \"privileged: true\"\n--    in the associated metadata.yaml file.\n\nDROP AGGREGATE IF EXISTS sg_jsonb_concat_agg(jsonb);",
```
Comment (camdencheek) [inline internal/database/migration/shared/data/stitched-migration-graph.json:9244]: Why this one wasn't caught by CI on the backport PR?

**Review by camdencheek — APPROVED** [review]

File: internal/database/migration/shared/data/stitched-migration-graph.json (hunk 1 above)
Comment (coury-clark) [reply internal/database/migration/shared/data/stitched-migration-graph.json:9244]: not sure - we've seen this happen before. Will continue to investigate

Comment (efritz) [reply internal/database/migration/shared/data/stitched-migration-graph.json:9244]: Because it didn't exist in the backport branch yet 👻

**Review by kalanchan — APPROVED** [review]

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.02% (+0.63 kb) | 0.03% (+4.69 kb) | 0.04% (+4.06 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/fixup-drift-from-50-release/compare-report.html;tab=live_object) for a full comparison between the commits fe0714d and 530863b or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: internal/database/migration/shared/data/stitched-migration-graph.json (hunk 1 above)
Comment (mucles) [reply internal/database/migration/shared/data/stitched-migration-graph.json:9244]: I am a bit confused here , wasn't this change already merged into `5.0` from this backport https://github.com/sourcegraph/sourcegraph/pull/49387

**Pull request merged**